scripts/build-python.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -eux
+
+ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")"/.. && pwd)
+cd "$ROOT_DIR"
+
+TMP_DIR="$1"
+rm -rf "$TMP_DIR"/dist
+
+./scripts/package-core.sh
+./scripts/package-torch.sh
+
+# check building sdist from a checkout, and wheel from the sdist
+python -m build python/metatensor-core --outdir "$TMP_DIR"/dist
+python -m build python/metatensor-operations --outdir "$TMP_DIR"/dist
+python -m build . --outdir "$TMP_DIR"/dist
+
+# for metatensor-torch, we need a pre-built version of metatensor-core, so
+# we use the one we just generated and make it available to pip
+dir2pi --no-symlink "$TMP_DIR"/dist
+
+PYPI_SERVER_PID=""
+function cleanup() {
+    kill $PYPI_SERVER_PID
+}
+# Make sure to stop the Python server on script exit/cancellation
+trap cleanup INT TERM EXIT
+
+python -m http.server --directory "$TMP_DIR"/dist 8888 &
```

Comment:
Maybe since this port is often busy with some jupyter notebook we do a check before?
```bash
PORT=8888
if [[ ! -z "$(netstat -nl | grep :$PORT)" ]]
then
    echo "An application is listening to port $PORT. Please free up the port first."  >&2
    exit 1
fi
```
https://serverfault.com/a/309111

Reply:
Very good point. I used `nc` since `netstat` is not installed everywhere, and I also changed the default port to something less likely to already be in use.